Python/pywarpx/picmi.py

```diff
@@ -2212,16 +2215,19 @@ class LabFrameFieldDiagnostic(picmistandard.PICMI_LabFrameFieldDiagnostic,
 
     warpx_upper_bound: vector of floats, optional
         Passed to <diagnostic name>.upper_bound
+
+    warpx_write_species: bool, optional, default=True
```

Comment:
In PICMI, I think it makes more sense to turn them off by default...
```suggestion
    warpx_write_species: bool, optional, default=False
```